make install with Julia v1.11.0-beta1  "Tarball with dependencies" hangs (Building HTML documentation)

Using the full tarball, that is "Tarball with dependencies", of Julia v1.11.0-beta1, the build hangs with the following message after issuing a `make install`: 

```
Building HTML documentation.
┌ Warning: Unable to determine HTML(edit_link = ...) from remote HEAD branch, defaulting to "master".
│ Calling `git remote` failed with an exception. Set JULIA_DEBUG=Documenter to see the error.
│ Unless this is due to a configuration error, the relevant variable should be set explicitly.
└ @ Documenter ~/Tmp/julia-1.11.0-beta1/doc/deps/packages/Documenter/2OZOh/src/utilities/utilities.jl:712
ERROR: LoadError: ArgumentError: Unable to automatically determine remote for main repo.
> `repo` is not set, and makedocs is not in a Git repository.
Configure `repo` and/or `remotes` appropriately, or set `remotes = nothing` to disable remote source
links altogether (e.g. if not working in a Git repository).
  repo: nothing
Stacktrace:
 [1] interpret_repo_and_remotes(; root::String, repo::String, remotes::Dict{String, Tuple{Documenter.Remotes.GitHub, String}})
   @ Documenter ~/Tmp/julia-1.11.0-beta1/doc/deps/packages/Documenter/2OZOh/src/documents.jl:658
 [2] interpret_repo_and_remotes
   @ ~/Tmp/julia-1.11.0-beta1/doc/deps/packages/Documenter/2OZOh/src/documents.jl:499 [inlined]
 [3] Documenter.Document(; root::String, source::String, build::String, workdir::Symbol, format::Documenter.HTMLWriter.HTML, clean::Bool, doctest::Bool, linkcheck::Bool, linkcheck_ignore::Vector{String}, linkcheck_timeout::Int64, checkdocs::Symbol, doctestfilters::Vector{Regex}, warnonly::Vector{Symbol}, modules::Vector{Module}, pages::Vector{Any}, pagesonly::Bool, expandfirst::Vector{String}, plugins::Vector{Documenter.Plugin}, repo::String, remotes::Dict{String, Tuple{Documenter.Remotes.GitHub, String}}, sitename::String, authors::String, version::String, highlightsig::Bool, draft::Bool, others::@Kwargs{})
   @ Documenter ~/Tmp/julia-1.11.0-beta1/doc/deps/packages/Documenter/2OZOh/src/documents.jl:429
 [4] Document
   @ ~/Tmp/julia-1.11.0-beta1/doc/deps/packages/Documenter/2OZOh/src/documents.jl:377 [inlined]
 [5] makedocs(; debug::Bool, format::Documenter.HTMLWriter.HTML, kwargs::@Kwargs{build::String, modules::Vector{Module}, clean::Bool, doctest::Bool, linkcheck::Bool, linkcheck_ignore::Vector{String}, checkdocs::Symbol, sitename::String, authors::String, pages::Vector{Any}, remotes::Dict{String, Tuple{Documenter.Remotes.GitHub, String}}})
   @ Documenter ~/Tmp/julia-1.11.0-beta1/doc/deps/packages/Documenter/2OZOh/src/makedocs.jl:242
 [6] top-level scope
   @ ~/Tmp/julia-1.11.0-beta1/doc/make.jl:374
in expression starting at /home/greg/Tmp/julia-1.11.0-beta1/doc/make.jl:374
make[1]: *** [Makefile:47: html] Error 1
make: *** [Makefile:120 : docs] Erreur 2

┌──(greg㉿ellipse)-[~/Tmp/julia-1.11.0-beta1]
```

It is the first time I encounter this issue. Maybe I should use the vanilla tarball to let the build process downloads dependencies and completes the build of the html documentation. A precision, my internet connection was functional.

--- a/Makefile
+++ b/Makefile
@@ -61,6 +61,10 @@ $(foreach link,base $(JULIAHOME)/test,$(eval $(call symlink_target,$(link),$$(bu
 julia_flisp.boot.inc.phony: julia-deps
 	@$(MAKE) $(QUIET_MAKE) -C $(BUILDROOT)/src julia_flisp.boot.inc.phony
 
+# Build the HTML docs (skipped if already exists, notably in tarballs)
+$(BUILDROOT)/doc/_build/html/en/index.html: $(shell find $(BUILDROOT)/base $(BUILDROOT)/doc \( -path $(BUILDROOT)/doc/_build -o -path $(BUILDROOT)/doc/deps -o -name *_constants.jl -o -name *_h.jl -o -name version_git.jl \) -prune -o -type f -print)
+	@$(MAKE) docs
+
 julia-symlink: julia-cli-$(JULIA_BUILD_MODE)
 ifeq ($(OS),WINNT)
 	echo '@"%~dp0/'"$$(echo '$(call rel_path,$(BUILDROOT),$(JULIA_EXECUTABLE))')"'" %*' | tr / '\\' > $(BUILDROOT)/julia.bat
@@ -272,7 +276,7 @@ define stringreplace
 endef
 
 
-install: $(build_depsbindir)/stringreplace docs
+install: $(build_depsbindir)/stringreplace $(BUILDROOT)/doc/_build/html/en/index.html
 	@$(MAKE) $(QUIET_MAKE) $(JULIA_BUILD_MODE)
 	@for subdir in $(bindir) $(datarootdir)/julia/stdlib/$(VERSDIR) $(docdir) $(man1dir) $(includedir)/julia $(libdir) $(private_libdir) $(sysconfdir) $(private_libexecdir); do \
 		mkdir -p $(DESTDIR)$$subdir; \
@@ -530,7 +534,7 @@ app:
 darwinframework:
 	$(MAKE) -C $(JULIAHOME)/contrib/mac/framework
 
-light-source-dist.tmp: docs
+light-source-dist.tmp: $(BUILDROOT)/doc/_build/html/en/index.html
 ifneq ($(BUILDROOT),$(JULIAHOME))
 	$(error make light-source-dist does not work in out-of-tree builds)
 endif

